etc/rc.d/Makefile

```diff
@@ -280,12 +280,6 @@ RCMDSPACKAGE=	rcmds
 FILES+=		routed
 .endif
 
-.if ${MK_SENDMAIL} != "no"
-FILESGROUPS+=	SMRCD
-SMRCD=		sendmail
-SMRCDPACKAGE=	sendmail
-.endif
-
```

Comment:
I am using sendmail a lot but there was an incomplete attempt to remove it at c0058945c6ef7b1f9b09b925d5a3470f9de66ce1 and this was forgotten